rspace/src/test/scala/coop/rchain/rspace/ScodecSerializationTests.scala

```diff
@@ -5,7 +5,7 @@ import coop.rchain.rspace.internal.scodecs._
 import org.scalatest.{FlatSpec, Matchers}
 import scodec.bits.ByteVector
 
-class ScodecSerializationTests extends FlatSpec with Matchers {
+trait ScodecSerializationTests extends FlatSpec with Matchers {
```

Comment:
For my own information, from now on am I supposed to use traits when I create tests under src/tests/ in general?

Reply:
Nope.  Here you've actually caught a mistake.  Fixing...

Where we do use traits in testing, it's generally because we composing several of them together to form a test class.